### ISSUE TITLE
Kissmanga not working

### ISSUE THREAD

**Issue by NhKPaNdA** [riderkick/FMD#509]
All my Kissmanga downloads give me an error. I tried going onto their website and it was working. When I tried to get into a manga chapter, it asked if I was human and gave me that "I am not a robot" check thing. After i checked it and resumed my errored Kissmanga downloads, they downloaded perfectly.

**Comment by kavin-90** [comment]
That is Cloudflare is doing which return noRecaptcha for wrong ip on its known list even if your ip is doing wrong stuff google will ask you for same thing.

**Comment by gliadon** [comment]
kavin-90, thank you! I've been having problems with Kissmanga regularly for the past few weeks. All manga downloads just stop with "Failed, try resuming this task!", just like Issue #496 , even though that was resolved. After you mentioned reCaptcha, I went to Kissmanga on my browser and opened a random chapter and it gave me a captcha. Once I completed it, I was able to download a few chapters with FMD. After downloading a few more chapters, I started getting the error again, but I was able to go complete a captcha in Chrome each time to temporarily clear up my IP, I guess. I suspect my IP was flagged for downloading too many files too fast, and now I have to aggressively convince them I'm not a robot, even though I kinda am by using FMD. I'm not sure that there's a way around this, other than maybe avoiding flagging by downloading at a slower rate? Is it possible to recognize a captcha check and allow it to be completed in FMD, instead of a separate browser?

tl;dr: If you're getting "Failed, try resuming this task!" on every chapter from Kissmanga (maybe other sites too), open a random chapter in your browser and complete the reCaptcha it gives you. This should allow you to download again for at least a little while.

**Comment by kavin-90** [comment]
For it i think you limit your task like run 1 task at a moment.

**Comment by riderkick** [comment]
There is nothing we can do right now. Kissmanga very hard protecting their stolen goods.
You know most of file you download from kissmanga actually comes from batoto. Just look at this image url:
`https://images1-focus-opensocial.googleusercontent.com/gadgets/proxy?container=focus&gadget=a&no_expand=1&resize_h=0&rewriteMime=image/*&url=http://img.bato.to/comics/2016/04/24/i/read571ce588a2444/img000001.jpg&imgmax=30000`
